content/en/integrations/guide/mongo-custom-query-collection.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Collect Mongo Custom Metrics
+kind: guide
+further_reading:
+- link: "/integrations/mongo/"
+  tag: "Documentation"
+  text: "Datadog-Mongo integration"
+---
+
+To collect custom metrics with the Datadog-Mongo integration, use the `custom_queries` option in the `conf.d/mongo.d/conf.yaml` file at the root of your [Agent's configuration directory][1]. See the sample [mongo.d/conf.yaml][2] for more details.
+
+## Configuration
+
+`custom_queries` has the following required options:
+
+* **`metric_prefix`**: Each metric starts with the chosen prefix.
+* **`query`**: This is the [Mongo runCommand][3] query to execute. Note that The Agent only supports `count`, `find` and `aggregates` queries.
+* **`field`**: Ignored for `count` queries. This is a list representing each field with no specific order. Unspecified and missing fields are ignored. There are 3 required pieces of data for each `field`:
+  * `field_name`: This is the name of the field from which to fetch the data.
+  * `name`: This is the suffix to append to the metric_prefix in order to form the full metric name. If `type` is `tag`, this column is instead considered as a tag that is applied to every metric collected by this particular query.
+  * `type`: This is the submission method (`gauge`, `count`, `rate`, etc..). This can also be set to `tag` to tag each metric in the row with the name and value of the item in this column. You can use the `count` type to perform aggregation for queries that return multiple rows with the same or no tags.
+
+The following optional options can also be used:
+
+* **`tags`**: A list of tags to apply to each metric (as specified above).
+* **`count_type`**: For `count` queries only, this is the submission method (`gauge`, `count`, `rate`, etc...) of the count result. Ignored for non count queries.
+
+## Examples
+
+For the examples below, the following mongo collection `user_collection` is used:
+
+```
+{ name: "foo", id: 12345, active: true, age:45, is_admin: true}
+{ name: "bar", id: 67890, active: false, age:25, is_admin: true}
+{ name: "foobar", id: 16273, active: true, age:35, is_admin: false}
+```
+
+Choose the type of query you would like to see an example for:
+
+{{< tabs >}}
+{{% tab "Count" %}}
+
+To monitor how many users are active at a given time for example. Your [Mongo count command][1] would be:
+
+```
+db.runCommand( {count: user_collection, query: {active:true}})
+```
+
+Which would correspond to the following `custom_queries` YAML configuration inside your `mongo.d/conf.yaml` file:
+
+```
+custom_queries:
+  - metric_prefix: mongo.users
+    query: {"count": "user_collection", "query": {"active":"true"}}
```

Comment:
I think we should have a comment somewhere to state that this is valid YAML. YAML is pure superset of JSON but this is not the most common knowledge, see https://yaml.org/spec/1.2/spec.html#id2759572

Reply:
I think this is outside our area of documentation as this is purely related to YAML specs and behaviors. Let's not overkill this doc :)

Reply:
Fair enough 👍